cmd/limactl/main.go

```diff
@@ -215,6 +217,34 @@ func handleExitCoder(err error) {
 	}
 }
 
+// executeWithPluginSupport handles command execution with plugin support.
+func executeWithPluginSupport(rootCmd *cobra.Command, args []string) error {
+	rootCmd.SetArgs(args)
+	err := rootCmd.Execute()
```

Comment:
I wonder if it wouldn't be cleaner to use `rootCmd.Find(...)` to see if the command is supported instead of parsing the error message. You may need to check extra for the `help` and `version` command that will be added by Cobra because they will not be part of the tree.

Reply:
I'll give it a try. I initially tried using `RunE` config in the root command to handle the unknown subcommands.
However, it didn't work well because Cobra checks for valid subcommands BEFORE calling it, that was why I used this current approach.